source/_patterns/components/view-selector/view-selector.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="pf-view-selector" role="group" aria-label="View selector">
```

Comment:
Does it make sense to use a btn group? http://v4-alpha.getbootstrap.com/components/button-group/